src/support/z_utils.erl

```diff
@@ -823,12 +796,6 @@ are_equal(_Arg1, _Arg2) ->
 
 
 %% @doc Return the name used in the context of a hostname
-%% @deprecated Please use the function {@link name_for_site/2} instead.
--spec name_for_host(Name :: atom(), atom() | #context{}) -> atom().
-name_for_host(Name, SiteOrContext) ->
-    name_for_site(Name, SiteOrContext).
-
-%% @doc Return the name used in the context of a hostname
 -spec name_for_site(Name :: atom(), atom() | #context{}) -> atom().
```

Comment:
The spec in line 799 uses a record, please define a type for the record and use that instead.